### ISSUE TITLE
`Readability.FleschReadingEase` not reported in Markdown files (or in the **PROBLEMS** pane)

### ISSUE THREAD

**Issue by thernstig** [ChrisChinchilla/vale-vscode#26]
We are getting the following issues when running the `vale` CLI via a terminal:

```sh
CONTRIBUTING.md:1:1:Readability.FleschReadingEase:Try to keep the Flesch reading ease score (61.04) above 70.
CONTRIBUTING.md:1:1:Readability.GunningFog:Try to keep the Gunning-Fog index (10.52) below 10.
CONTRIBUTING.md:1:1:Readability.LIX:Try to keep the LIX score (35.33) below 35.
CONTRIBUTING.md:1:1:Readability.SMOG:Try to keep the SMOG grade (11.15) below 10.
CONTRIBUTING.md:1:1:Readability.FleschKincaid:Try to keep the Flesch–Kincaid grade level (8.23) below 8.
CONTRIBUTING.md:1:1:Readability.ColemanLiau:Try to keep the Coleman–Liau Index grade (9.01) below 9.
```
They are not visible in the **PROBLEMS**  pane, nor as actual yellow squiggly lines in the editor.

**PROBLEMS pane**
![image](https://github.com/ChrisChinchilla/vale-vscode/assets/30827238/78a19f70-57f1-43c9-a4f6-f2d773778d19)

However, they are for some reason visible in the status bar. Which is not very visible and seems like a strange placement:
![image](https://github.com/ChrisChinchilla/vale-vscode/assets/30827238/c771eb2a-794f-4ba1-997d-db0356f1ca89)

Should they not be visible in the **PROBLEMS** pane and in the actual markdown files?

**Comment by ChrisChinchilla** [comment]
@thernstig If you look in the settings, you can change the position of the readability rules.

**Comment by thernstig** [comment]
@ChrisChinchilla fair enough, thanks! But I do think the default should change to be showed in the problems pane. At least, I find it very strange that it is shown in the status bar. It clutters it quite a bit and is not a normal location for any other extension afaik to place information/errors.
